src/planner/WorldStateSaver.cpp

```diff
@@ -0,0 +1,19 @@
+#include <aikido/planner/WorldStateSaver.hpp>
+
+namespace aikido {
+namespace planner {
+
+WorldStateSaver::WorldStateSaver(
+    WorldPtr world)
+  : mWorld{world}
```

Comment:
Would it make sense to initialize `mWorldState` here instead?

Reply:
`WorldState` does not keep a pointer to a specific world, but when `WorldStateSaver` is destroyed, we typically want to restore the state of a `World` to a previous state. So I think we need to keep a pointer to the world?